src/main/java/de/mossgrabers/convertwithmoss/format/yamaha/ysfc/YamahaMOXFEWIMEntry.java

```diff
@@ -0,0 +1,45 @@
+package de.mossgrabers.convertwithmoss.format.yamaha.ysfc;
```

Comment:
Same as above: To not pollute the main code with changes to this block, the entry for the section was subclassed. 

Not an ideal solution from theoretical point of view, but works.